docs/src/guide/tutorial.jl

```diff
@@ -42,8 +51,8 @@ atoms = [Si => [ones(3)/8, -ones(3)/8]]
 
 ## 2. Select model and basis
 model = model_LDA(lattice, atoms)
-kgrid = [4, 4, 4]  # k-point grid (Regular Monkhorst-Pack grid)
-Ecut = 7           # kinetic energy cutoff in Hartree
+kgrid = [4, 4, 4]      # k-point grid (Regular Monkhorst-Pack grid)
+Ecut = 190.5u"eV"  # kinetic energy cutoff
```

Comment:
Just make sure that the hashes are aligned ;) looks a little nicer.

Reply:
It's very unusual to use eV for Ecuts. I would leave it as `7` and add "in Hartree. Can also be specified with the Unitful package, see above`

Reply:
That is not true. eV is the default unit for ASE.

Reply:
No DFT code I know works in eV for Ecuts, and for good reason: it's not the natural energy scale to use for this, because Ecut has to be big compared to the characteristic kinetic energy of the electrons, which is much more naturally in Hartree than in eV (as seen here). OTOH things people commonly report as *outputs*, eg band plots or such, are about differences of energies that are much smaller than that, and for this Hartree makes little sense (and indeed we plot them in eV)

Reply:
> No DFT code I know works in eV for Ecuts

[VASP](https://www.vasp.at/wiki/index.php/ENCUT) does and CASTEP, too. Probably there are some more.  
I completely agree that eV is a stupid unit for this, but you'd be surprised, but it's not that uncommon as you think. Many people in the physics community think in terms of eV for cutoffs from the interactions I had. We should definitely have this in the tutorial to make people aware.

Reply:
Oh wow I had no idea, sorry! That's super surprising, all the talk about convergence I've seen is in Hartree. I still don't think we should have it as the default (it's not a habit we want to encourage), but yeah OK it makes a lot of sense to be super explicit. Maybe something like `Ecut = 7 # in Hartree, equivalent to eg `Ecut = 190.5u"eV"``?

Reply:
Hmm, perhaps more like:
```julia
Ecut = 7  # in Hartree
## Ecut = 190.5u"eV"  # Could also use eV to specify cutoffs
```